### PR TITLE
Find Forms: Working on a fix for safari Screen reader.

### PR DESCRIPTION
**Pull request by zacharymorel**
## Description
Screen reader on Safari was counting the numbers of nodes nested within the header.
Needed to prevent that counting.
104336907-e8bcad80-54c2-11eb-8fe5-84e918c2c276

## Testing done
Ran e2e, unit, manually tested mac os voice over on Chrome, FF, Safari.

## Screenshots
Screen Shot 2021-02-24 at 11 37 35 AM

## Acceptance criteria
- [x] The screen reader does not count the number of elements within the text results element.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/find-forms/containers/SearchResults.jsx
Comment (joshkimux) [inline src/applications/find-forms/containers/SearchResults.jsx:185]: @zacharymorel , question for you. Would it be possible to have an empty aria-live region div on the page _before_ the user searches for a form? If the `h2` search result is added into an already existing empty aria-live region, `aria-live="polite"` _should_ work. Otherwise, it may hiccup or act unpredictably as you described earlier.

> The ARIA live region must be present and must be empty before making an announcement. The best approach is to ensure the ARIA live region is already on the page when the page loads. You cannot insert an ARIA live region and make an announcement at the same time. The region needs to be on the page, and the accessibility API needs to be aware that it is there and that it is empty before any announcements will be sent to the screen reader. If you insert the ARIA live region after page load, you must use JavaScript to delay (typically for at least one second) before sending any text into it to make an announcement.

Apologies in advance if I read the code wrong and this behavior has already been tested 🙇

Comment (zacharymorel) [reply src/applications/find-forms/containers/SearchResults.jsx:185]: Hey @joshkimux have you tested this locally? I think what we have works decent on both VO safari and chrome?

Comment (joshkimux) [reply src/applications/find-forms/containers/SearchResults.jsx:185]: @zacharymorel I don't have access just yet as I still have to set up SOCKS (doing this next week), so as of now I've only been able to review the code. Is there any chance you could pass this through staging for me to audit?

Comment (zacharymorel) [reply src/applications/find-forms/containers/SearchResults.jsx:185]: Yes, I can merge this after @kelsonic || @ncksllvn approval.

**Review by kelsonic — APPROVED** [review]
LGTM 🎉

**Pull request merged**
